azure-pipelines.yml

```diff
@@ -28,91 +28,92 @@ jobs:
     name: Linux
     vmImage: ubuntu-16.04
     matrix:
-      py37_np115_cov:
+      py37_np115_32bit:
+        # 32 bit linux only has np 1.15
         PYTHON: '3.7'
         NUMPY: '1.15'
         CONDA_ENV: azure_ci
-        RUN_COVERAGE: yes
-        RUN_FLAKE8: yes
-        RUN_MYPY: yes
+        BITS32: yes
         TEST_START_INDEX: 2
-      py37_np115_32bit:
+      py37_np117_cov:
```

Comment:
Why is the ordering swapped here? From cov first to 32-bit first?

Reply:
I swapped them as they indexed in vaguely lexicographical order, `py37_np115` < `py37_np117`, however... I'm not sure why this is still NumPy 1.15 for 32 bit and how CI is actually managing to get NumPy 1.17 from pip. Suggest we don't merge this until this is figured out.